src/styles/main.scss

```diff
@@ -1,3 +1,60 @@
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  height: 100vh;
+  padding: $padding;
+  margin: 0 auto;
+  width: ($day-box-size + $gap-between-boxes) * $columns + ($padding * 2);
+  list-style: none;
+
+  display: flex;
+  align-items: center;
+  align-content: center;
+  gap: $gap-between-boxes;
+  flex-wrap: wrap;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background-color: #eee;
+    width: $day-box-size;
+    height: $day-box-size;
+    border: 1px solid;
+
+    cursor: pointer;
+    transition: all 0.5s;
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+    }
+    @for $numberOfDay from 1 through 31 {
+      &:nth-child(#{$numberOfDay})::before {
+        content: "#{$numberOfDay}";
+      }
+    }
+  }
+
+  @for $month-length from 28 through 31 {
+    &--month-length--#{$month-length} :nth-child(n + #{$month-length + 1}) {
+      display: none;
+    }
+  }
+
+  @each $day, $num in $start-day {
+    &--start-day--#{$day} :first-child {
+      margin-left: ($day-box-size + $gap-between-boxes) * $num;
```

Comment:
variable for this

Reply:
@tiserett I don't know how to deal with this. Couse $num depends on the @each cycle. How could i do that?